src/app/cart/cart.component.js

```diff
@@ -29,29 +29,22 @@ class CartController {
   }
 
   loadCart() {
+    this.loading = true;
     this.cartService.get()
       .subscribe( ( data ) => {
         this.cartData = data;
+        this.loading = false;
       } );
   }
 
-  removeItem( uri ) {
```

Comment:
Wasn't this being used to delete items from the cart, not just updating? The delete button seems to be broken now.